.github/workflows/prerelease.yml

```diff
@@ -0,0 +1,41 @@
+name: PreRelease
+
+on: 
+  workflow_dispatch:
+
+jobs:
+  Publish:
+    name: PreRelease
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout
+        uses: actions/checkout@v2
+        with:
+          token: ${{ secrets.ADMIN_TOKEN }}
+          ref: ${{ github.ref }}
+          # pulls all commits (needed for lerna / semantic release to correctly version)
+          fetch-depth: '0'
```

Comment:
if think we don't need it here, since we don't care about history if we don't produce a changelog

Reply:
removed